releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2266,7 +2266,7 @@
   "124417": {
     "commit": "74877bb43bc88e21b7304382cf039947a7d90069",
     "text": "The feature gate \"DefaultHostNetworkHostPortsInPodTemplates\" has been removed.  This behavior was deprecated in v1.28, and has had no reports of issues since.",
-    "markdown": "The feature gate \"DefaultHostNetworkHostPortsInPodTemplates\" has been removed.  This behavior was deprecated in v1.28, and has had no reports of issues since. ([#124417](https://github.com/kubernetes/kubernetes/pull/124417), [@thockin](https://github.com/thockin)) [SIG Apps]",
+    "markdown": "The feature gate \"DefaultHostNetworkHostPortsInPodTemplates\" has been removed.  This behavior was deprecated in v1.28, and has had no reports of issues since. ([#124417](https://github.com/kubernetes/kubernetes/pull/124417), [@thockin](https://github.com/thockin))",
```

Comment:
```suggestion
    "markdown": "The feature gate \"DefaultHostNetworkHostPortsInPodTemplates\" has been removed.  This behavior was deprecated in v1.28, and has had no reports of issues since. ([#124417](https://github.com/kubernetes/kubernetes/pull/124417), [@thockin](https://github.com/thockin)) [SIG Apps]",
```